### PR TITLE
Update Help Command

### LINKED ISSUE #204: Update Help Command
Include new commands and update any old commands

### PR DESCRIPTION
**Pull request by jicsontoh**

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/HelpCommand.java [hunk 1]
```diff
@@ -18,19 +18,19 @@ public class HelpCommand extends Command {
 
     // All commands summary
     public static final String ALL_COMMANDS_MESSAGE = "Here are all the valid commands:\n"
```
Comment (ElginL) [inline src/main/java/seedu/address/logic/commands/HelpCommand.java:20]: Nice, I like the addition of indexes

File: src/main/java/seedu/address/logic/commands/HelpCommand.java [hunk 2]
```diff
@@ -18,19 +18,19 @@ public class HelpCommand extends Command {
...
-            + "help\n"
-            + "list\n"
-            + "user_guide\n"
-            + "clear";
+            + "1. add n/CLIENT a/ADDRESS p/PHONE e/EMAIL [t/TAG]\n"
+            + "2. buy INDEX q/QUANTITY g/GOODS price/PRICE\n"
+            + "3. sell INDEX q/QUANTITY g/GOODS price/PRICE\n"
+            + "4. edit INDEX [n/NAME] [a/ADDRESS] [p/PHONE] [e/EMAIL] [t/TAG]\n"
+            + "5. view INDEX\n"
+            + "6. delete INDEX\n"
```
Comment (ElginL) [inline src/main/java/seedu/address/logic/commands/HelpCommand.java]: This is a wrong Delete command usage. It now has a flag that is either `m/transaction`, `m/client`, or `m/remark`. Its the same for edit, but its okay, I can change it tmr

**Review by ElginL — APPROVED** [review]
LGTM

File: src/main/java/seedu/address/logic/commands/HelpCommand.java [hunk 3]
```diff
@@ -18,19 +18,19 @@ public class HelpCommand extends Command {
...
-            + "find KEYWORD\n"
-            + "exit\n"
-            + "help\n"
-            + "list\n"
-            + "user_guide\n"
-            + "clear";
+            + "1. add n/CLIENT a/ADDRESS p/PHONE e/EMAIL [t/TAG]\n"
+            + "2. buy INDEX q/QUANTITY g/GOODS price/PRICE\n"
+            + "3. sell INDEX q/QUANTITY g/GOODS price/PRICE\n"
+            + "4. edit INDEX [n/NAME] [a/ADDRESS] [p/PHONE] [e/EMAIL] [t/TAG]\n"
```
Comment (quannam0124) [inline src/main/java/seedu/address/logic/commands/HelpCommand.java]: This edit command now includes another tag after INDEX which is either m/client m/transaction, or m/remark. I can edit this as well

**Review by quannam0124 — APPROVED** [review]
LGTM! I can help edit the command message for edit in my PR

**Review by notle1706 — CHANGES_REQUESTED** [review]
I think it would be better to change the things that @ElginL @quannam0124 pointed out.

File: src/main/java/seedu/address/logic/commands/HelpCommand.java (hunk 1 above)
Comment (jicsontoh) [reply src/main/java/seedu/address/logic/commands/HelpCommand.java:20]: thank you!

File: src/main/java/seedu/address/logic/commands/HelpCommand.java (hunk 2 above)
Comment (jicsontoh) [reply src/main/java/seedu/address/logic/commands/HelpCommand.java]: no worries, will update this. Was waiting to merge the new delete/edit before updating the command!

File: src/main/java/seedu/address/logic/commands/HelpCommand.java (hunk 3 above)
Comment (jicsontoh) [reply src/main/java/seedu/address/logic/commands/HelpCommand.java]: no worries, will push another commit to update!

**Pull request merged**